### ISSUE TITLE
Jugend centers in Austria

### ISSUE THREAD

**Issue by brodecva** [odalic/sti#176]
https://grips.semantic-web.at/x/sASBB

**Comment by josef-janousek** [comment]
Version: develop 11.12.2016
File: http://data.ooe.gv.at/files/cms/Mediendateien/OGD/ogd_dirBGD/Jugendzentren.csv

Task execution ended by error. The UI displayed the error statement, but user cannot recognize what is the reason of the error:
![jugendzentren_error](https://cloud.githubusercontent.com/assets/17772934/21080718/7d578ed4-bfb6-11e6-99b8-07da192ec4e6.png)

I found that the error was caused by SPARQLProxy and the method queryEntityCandidates when the query string contains quotes, In the original file in the first column some rows contain string with quotes, for example "Girls-Treff ""Insel""" in row 6 or "Jugendzentrum ""Ins Schwarze""" in row 17. These strings are problematic for this query method.
